processes/sfrCluster.py

```diff
@@ -101,7 +101,7 @@ def clusterRecord(self, rec):
 
         try:
             self.session.flush()
-        except (DataError, StaleDataError) as e:
+        except (IntegrityError, DataError, StaleDataError) as e:
```

Comment:
Hmm - what would happen if one of these errors are not thrown? Should we generally handle all exceptions here?

Reply:
Awesome, that sounds good!